frontend/app/src/pages/balances/blockchain/[[tab]].vue

```diff
@@ -43,6 +46,21 @@ const { unifyAccountsTable } = storeToRefs(useFrontendSettingsStore());
 const { supportedChains } = useSupportedChains();
 const { groups } = storeToRefs(useBlockchainStore());
 
+function assetFilter(item: AssetBalance) {
+  const keyword = get(search).toLocaleLowerCase()?.trim() ?? '';
+
+  if (!keyword)
+    return true;
+
+  const name = get(assetName(item.asset))?.toLocaleLowerCase()?.trim();
+  const symbol = get(assetSymbol(item.asset))?.toLocaleLowerCase()?.trim();
+  return symbol.includes(keyword) || name.includes(keyword);
+}
+
+const filteredBlockchainAssets = computed(() =>
+  get(blockchainAssets).filter(assetFilter),
+);
+
```

Comment:
wouldn't it make more sense to move the filtering itself in `AssetFilter` and use keyword as a `v-model:search`?